README.md

```diff
@@ -1,5 +1,12 @@
+импорты common.js, работает на node.js
+
+чтобы посмотреть, как все работает, можно зайти в test.js (там есть примеры вызовов и тд)
```

Comment:
Well, actually your `test.js` is not what gnerally called tests. I'd rather name it simply `index.js`

Reply:
fixed